neqo-bin/src/server/http3.rs

```diff
@@ -127,9 +123,9 @@ impl super::HttpServer for HttpServer {
                     } else if let Ok(count) =
                         path.value().trim_matches(|p| p == '/').parse::<usize>()
                     {
-                        ResponseData::repeat(Self::MESSAGE, count)
+                        ResponseData::zeroes(count)
                     } else {
-                        ResponseData::from(Self::MESSAGE)
+                        ResponseData::from(path.value())
```

Comment:
If the request was neither from a QNS test, nor was it a number, requesting a certain response size, we were previously sending 32k zeroes. Instead, we now simply echo the path. 

I assume the previous was arbitrary. I am fine reverting to the previous version.